docs/developer-resources/guides/first-app-v3/theming.md

```diff
@@ -22,7 +22,7 @@ $colors: (
 
 But wait, there’s more! Ionic automatically provides platform specific styles based on the device the application is running on, giving that native look and feel your users are used to:
 
-![ios and android comparison](/img/guides/first-app-v3/ion-lab-comparison.png)
+![Screenshots comparing the native look of an Ionic app on iOS and Android platforms.](/img/guides/first-app-v3/ion-lab-comparison.png 'Ionic Platform Specific Styles Comparison')
```

Comment:
This one also needs to updated to exclude "screenshot". The same goes for the same sentences found in other files.